config/config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "development": {
```

Comment:
Do not hardcode the values here. Instead, use environment variables. 

- https://www.codementor.io/teodeleanu/how-to-use-sequelize-orm-in-your-express-application-u5d78po6f
- https://sequelize.org/master/manual/migrations.html